client/source/js/modules/ui/index.js

```diff
@@ -15,7 +15,7 @@ define([
     'app.ui.menu'
   ])
 
```

Comment:
oh and by the way, the card required to add buttons into create/manage project screen, not to the side menu.